cli/src/main/java/com/box/l10n/mojito/cli/command/DropViewCommand.java

```diff
@@ -0,0 +1,172 @@
+
+    /**
+     * logger
+     */
+    static Logger logger = LoggerFactory.getLogger(DropViewCommand.class);
+
+    @Autowired
+    ConsoleWriter consoleWriter;
+
+    @Parameter(names = {Param.REPOSITORY_LONG, Param.REPOSITORY_SHORT}, arity = 1, required = true, description = Param.REPOSITORY_DESCRIPTION)
+    String repositoryParam;
+
+    @Parameter(names = {"--number-drops-fetched"}, arity = 1, required = false, description = "Number of drops fetched")
+    Long numberOfDropsToFetchParam = 10L;
+
+    @Parameter(names = {"--show-all", "-all"}, required = false, description = "Show all drops (already imported drops are hidden by default)")
+    Boolean alsoShowImportedParam = false;
+
+    @Parameter(names = {"--csv"}, required = false, description = "When showing drops, output info in csv format so that it is easily parsable by scripts.")
+    Boolean showCsvOutput = false;
+
+    @Parameter(names = {"--json"}, required = false, description = "When showing drops, output info in json format so that it is easily parsable by scripts.")
+    Boolean showJsonOutput = false;
+
+    @Parameter(names = {"--dropid", "-i"}, required = false, arity = 1, description = "Show information only about the drop with the given id")
+    Long dropid = -1L;
+
+    @Autowired
+    CommandHelper commandHelper;
+
+    @Autowired
+    Console console;
+
+    @Autowired
+    DropClient dropClient;
+
+    private void outputInfo(Drop drop) {
+        if (dropid != -1 && drop.getId().longValue() != dropid.longValue()) return;
+        if (showCsvOutput) {
+            consoleWriter.a(drop.getId()).a(",").a(drop.getName());
+
+            if (Boolean.TRUE.equals(drop.getCanceled())) {
+                consoleWriter.a(",CANCELED");
+            } else if (drop.getLastImportedDate() == null) {
+                consoleWriter.a(",NEW");
+            } else {
+                consoleWriter.a(",").a(drop.getLastImportedDate());
+            }
+        } else {
+            consoleWriter.
+                    a("id: ").fg(Color.MAGENTA).a(drop.getId()).reset().
+                    a(", name: ").fg(Color.MAGENTA).a(drop.getName()).reset();
+
+            if (Boolean.TRUE.equals(drop.getCanceled())) {
+                consoleWriter.fg(Color.GREEN).a(" CANCELED");
+            } else if (drop.getLastImportedDate() == null) {
+                consoleWriter.fg(Color.GREEN).a(" NEW");
+            } else {
+                consoleWriter.a(", last import: ").fg(Color.MAGENTA).a(drop.getLastImportedDate());
+            }
+        }
+        consoleWriter.println();
+    }
+
+    @Override
+    public void execute() throws CommandException {
+
+        Repository repository = commandHelper.findRepositoryByName(repositoryParam);
+
+        Map<Long, Drop> numberedAvailableDrops = getNumberedAvailableDrops(repository.getId());
+
+        if (numberedAvailableDrops.isEmpty()) {
+            consoleWriter.newLine().a("No drop available").println();
```

Comment:
If we are treating "no drops available" outcome as valid (i.e. exit code 0), I think we should also provide valid outputs for CSV and JSON modes so that parsing it would result in valid (albeit empty) dataset - for CSV that would probably be just empty string or newline and for JSON an empty array

Reply:
changed as requeted